docs/topics/feed-exports.rst

```diff
@@ -267,10 +267,140 @@ once all the file contents have been written (i.e. at the end of the crawl) is
 that file uploaded to the feed URI.
 
 If you want item delivery to start earlier when using one of these storage
-backends, use :setting:`FEED_EXPORT_BATCH_ITEM_COUNT` to split the output items
-in multiple files, with the specified maximum item count per file. That way, as
-soon as a file reaches the maximum item count, that file is delivered to the
-feed URI, allowing item delivery to start way before the end of the crawl.
+backends, use :ref:`batches<batches>` to split the output items in multiple
+files, based on a specified limit. That way, as soon as a file reaches the specified
+limit, that file is delivered to the feed URI, allowing item delivery to start
+way before the end of the crawl.
+
+.. _batches:
+
+Batches
+=======
+
+.. versionadded:: VERSION
+
+Batches enable multiple sequential files to be generated for a feed instead of getting
+a single file. This is useful in getting item deliveries earlier before the crawl has
+been ended.
+
+To enable batching, you can use :ref:`criterias<batch_criterias>` such as ``batch_item_count``
+, ``batch_file_size``, ``batch_duration`` or you can create :ref:`your own batch
+handlers<custom_batch_handler>` to handle batch deliveries.
```

Comment:
This is written from the perspective of a user that is using the default batch handler. While this will be true for most users, the documentation should be accurate.

What about:

```suggestion
Batching is controlled by a batch handler, which is defined through the 
:ref:`batch feed option <feed-options>`.

For information about enabling and configuring the default batch handler, see
:class:`~scrapy.extensions.batches.BatchHandler`. You can alternatively 
:ref:`create your own batch handler <custom_batch_handler>`.
```

Then you could create a section called something like “Built-in Batch Handler”, and use `.. autoclass:: scrapy.extensions.batches.BatchHandler` there to provide documentation about the handler directly from its docstring. Then you can place in its docstring all the information about enabling and configuring the default batch handler, including what you currently have under the “Batch Criterias” section below (by the way, it’s “criteria” even in plural), without implying that every batch handler is supposed to be enabled and configured the same way.